packages/@n8n/api-types/src/dto/insights/list-workflow-query.dto.ts

```diff
@@ -30,7 +32,8 @@ const sortByValidator = z
 	.optional();
 
 export class ListInsightsWorkflowQueryDto extends Z.class({
-	...paginationSchema,
+	skip: skipValidator,
```

Comment:
Would it make sense to include the full paginationSchema and just overwrite the take validator? For example:

```
 ...paginationSchema,
take: createTakeValidator(MAX_ITEMS_PER_PAGE),
```

This way if the paginationSchema changes otherwise, we will automatically get these changes.